lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -0,0 +1,7 @@
+<div class="certificate-container">
+    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <img src="/static/images/xseries-certificate-visual.png" alt="XSeries Program Certificates" aria-hidden="true">
```

Comment:
@AlasdairSwan Regarding your question about the image src attribute, I think the rule I have used is: If the src value is dynamic or comes from outside of the edx-platform code base, I would pipe it in through the backend. If its a static image within the edx-platform, I would just code the path into the element. That being said, the way this image src is written is wrong I think. Please look at https://github.com/edx/edx-platform/blob/master/lms/templates/dashboard/_dashboard_course_listing.html#L83
Notice it uses a `static.url()` function